3rdParty/jemalloc/CMakeLists.txt

```diff
@@ -31,8 +31,14 @@ if (LINUX OR DARWIN)
     set(JEMALLOC_CONFIG "background_thread:true,cache_oblivious:false")
   endif ()
 
+  SET(JEMALLOC_C_FLAGS ${CMAKE_C_FLAGS})
   if (USE_JEMALLOC_PROF)
-    SET(JEMALLOC_PROF "--enable-prof")
+      if (USE_LIBUNWIND)
+          SET(JEMALLOC_PROF "--enable-prof" "--enable-prof-libunwind" "--with-static-libunwind=${LIBUNWIND_LIB}")
+          SET(JEMALLOC_C_FLAGS "${CMAKE_C_FLAGS} -I${LIBUNWIND_HOME}/include")
+      else ()
+          SET(JEMALLOC_PROF "--enable-prof")
+      endif()
   endif ()
```

Comment:
I think set to the CMAKE_C_FLAGS will be ok, because we're in jemalloc cmakelists.txt.
But it doesn't really matter